bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/servlets/MultiFieldDropTargetPostProcessor.java

```diff
@@ -119,6 +137,20 @@ private void processProperty(Resource resource, String target, String propertyVa
             resourceResolver.delete(dropTargetResource);
         }
 
+        if (selectionType != null) {
+            if (SKU.equals(selectionType) || COMBINED_SKU.equals(selectionType)) {
+                propertyValue = StringUtils.substringAfterLast(propertyValue, "/");
+            } else if (SLUG.equals(selectionType)) {
+                Resource product = resourceResolver.getResource(propertyValue);
```

Comment:
The SLUG resolving part must be changed, this only works because of the virtual product bundles are installed by the connector. We must not have any dependency on virtual products within the components project. This is will also very likely break with the new authoring tools.

So maybe two simple reasons to move the entire post-processor to the connector project. That is IMHO the better place since it does something which is related to the authoring tools. Then this code part would also be ok.

Reply:
I agree it's something more related to authoring so we can move it in the connector project. But strictly speaking, this doesn't really remove the dependency, because for example the `carousel` DnD target would still be defined with `./multiDropTarget->@product`, so it wouldn't work without the connector.

Note that we could also simply remove the "slug" option: both the carousel and the teaser anyway do not support having product(s) configured by slug, so I only added it to demonstrate it's possible to support that, but we don't have any use case for this.